ert/ensemble_evaluator/builder/_ensemble.py

```diff
@@ -76,6 +81,23 @@ async def send_cloudevent(  # pylint: disable=too-many-arguments
         assert client.websocket  # mypy
         await client.websocket.close()
 
+    # TODO: make legacy-only?
+    # See https://github.com/equinor/ert/issues/3456
+    @property
+    @abstractmethod
+    def output_bus(
+        self,
+    ) -> "asyncio.Queue[CloudEvent]":
+        raise NotImplementedError
+
+    # TODO: make legacy-only?
+    # See https://github.com/equinor/ert/issues/3456
+    async def queue_cloudevent(
```

Comment:
Can we call this `add_cloudevent` since it sounds to me that we want handle to the queue only instead, but it might be just me - so leave it up to you :)